tox.ini

```diff
@@ -10,15 +10,12 @@ passenv =
 # usedevelop = true
 skip_install = true
 # NOTE: environment variables *MUST* include the `==`, e.g.,
-#       SPHINX_VERSION="==3.0.0" tox -e py
+#       SPHINX_VERSION="==X.Y.Z" tox -e py
 # This way, if the variable is not set, it uses the latest version.
 deps =
-    # TODO: drop this when sphinx 3.x is dropped.
-    # https://github.com/sphinx-doc/sphinx/issues/10291
-    Jinja2<3.1
     # requirements for exhale
-    sphinx{env:SPHINX_VERSION:}
-    breathe{env:BREATHE_VERSION:>=4.32.0}
+    sphinx{env:SPHINX_VERSION:>=4.3.2}
```

Comment:
blocking myself in no particular location, update the readme in this PR too https://github.com/svenevs/exhale#exhale-version-compatibility-with-python-sphinx-and-breathe